### PR TITLE
Signare lab proposal

### PR DESCRIPTION
**Pull request by coeniebeyers**
# Signare
Signare, a Hyperledger Lab, is an enterprise grade digital signing solution for DLT related applications and Ethereum client

### CODE REVIEW

File: labs/hyperledger-signare.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Signare
+parent: Labs
+---
+# Lab Name
+
+Hyperledger Signare
```
Comment (tkuhrt) [inline labs/hyperledger-signare.md:8]:
```suggestion
Signare
```

Labs cannot use Hyperledger as a prefix.

File: labs/hyperledger-signare.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Short Description
+
+Signare, a Hyperledger Lab, is an enterprise grade digital signing solution for DLT related applications and Ethereum clients.
+
+# Scope of Lab
+
+A security concern shared by most users of DLT applications is "keeping their private key private". In the enterprise space FIPS 140 is often used to inform institutions of how they must manage their private keys. Specifically, FIPS 140-2 Level 2 adds requirements for physical tamper-evidence (and/or tamper-resistance) and role-based authentication, which necessitates the use of an HSM or Cloud HSM. 
+
+The purpose of Hyperledger Signare is to provide a FIPS 140-2 Level 2 compliant signing solution for enterprise applications where various HSM and Cloud HSM vendors will be supported via plugins. Hyperledger Signare also provides role-based access controlled interfaces to solve multiple usecases, such as signing Ethereum transactions and for blockchain clients such as Hyperledger Besu to store keys in an HSM or Cloud HSM.
```
Comment (tkuhrt) [inline labs/hyperledger-signare.md:18]:
```suggestion
The purpose of Signare is to provide a FIPS 140-2 Level 2 compliant signing solution for enterprise applications where various HSM and Cloud HSM vendors will be supported via plugins. Signare also provides role-based access controlled interfaces to solve multiple usecases, such as signing Ethereum transactions and for blockchain clients such as Hyperledger Besu to store keys in an HSM or Cloud HSM.
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Looks like a good lab. Minor changes requested, but then I will approve.

**Review by alvaropicazo — APPROVED** [review]

File: labs/hyperledger-signare.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+https://github.com/Jserrano27
+https://github.com/gynura
+https://github.com/ArturoGarciaRegueiro
+https://github.com/mkrielza
+https://github.com/coeniebeyers
+
+
+# Sponsor
+ 
+- Susumu Toriumi (susumu.toriumi@datachain.jp) - Maintainer, YUI
```
Comment (nidhi-singh02) [inline labs/hyperledger-signare.md:34]: Can you please add Github of the sponsor.

**Review by tkuhrt — APPROVED** [review]

File: labs/hyperledger-signare.md (hunk 3 above)
Comment (tkuhrt) [reply labs/hyperledger-signare.md:34]: @storiumi : Can you confirm sponsorship?

Comment (storiumi) [reply labs/hyperledger-signare.md:34]: @tkuhrt (cc. @coeniebeyers ) Confirmed. Happy to support the lab.

**Review by nidhi-singh02 — APPROVED** [review]
lgtm! Thank you @coeniebeyers

**Comment by coeniebeyers** [comment]
Thanks so much everyone!

**Comment by tkuhrt** [comment]
https://github.com/hyperledger-labs/governance/pull/72

**Comment by tkuhrt** [comment]
https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/273

**Comment by tkuhrt** [comment]
Repo created: https://github.com/hyperledger-labs/signare - Please look for invitations to the hyperledger-labs organization

**Comment by davidwboswell** [comment]
@coeniebeyers -- If you'd like some help promoting your new lab to the Hyperledger community, I'm happy to help.  We can organize a virtual meetup and share that -- when we've done that for other labs we get 100-200 people signing up for a meetup and then we see hundreds of people watching the recording of that on our YouTube channel.  Feel free to reach out to me at dboswell at linuxfoundation dot org if you'd like to set up something.

**Comment by coeniebeyers** [comment]
> @coeniebeyers -- If you'd like some help promoting your new lab to the Hyperledger community, I'm happy to help. We can organize a virtual meetup and share that -- when we've done that for other labs we get 100-200 people signing up for a meetup and then we see hundreds of people watching the recording of that on our YouTube channel. Feel free to reach out to me at dboswell at linuxfoundation dot org if you'd like to set up something.

Hi @davidwboswell thanks so much for reaching out! We were discussing internally when best to do something like this, we do have a potential Signare Webinar taking place on the 24th of July, does it make sense to perhaps combine that with a virtual meetup?

**Comment by davidwboswell** [comment]
@coeniebeyers -- Great, glad to hear you're interested in a meetup.  Yes, we could potentially combine this with a webinar.  Note that we tend to see lower turnout for our online meetups in late July and August, so we may want to consider doing something in September.  That would also give us more time to promote this.  Let's find some time to talk and we can sort out the details.  Does Friday, July 5 at 9:30 pacific or Monday, July 8 at 7:30, 8, 8:30, 10, 10:30 or 11 pacific work for you for a call?  If so, let me know and send me your email and I'll send out an invitation.

**Pull request merged**